drivers/usb/dwc2/platform.c

```diff
@@ -439,6 +439,9 @@ static int dwc2_driver_probe(struct platform_device *dev)
 	struct dwc2_hsotg *hsotg;
 	struct resource *res;
 	int retval;
+	const char *compat;
+	int ret;
```

Comment:
Where do you use ret and why not use retval?
P.S. Please always make sure that your code doesn't produce any build warnings